gobblin-cluster/src/test/java/org/apache/gobblin/cluster/FsJobConfigurationManagerTest.java

```diff
@@ -91,6 +91,10 @@ public void setUp() throws IOException {
       return null;
     }).when(this.eventBus).post(Mockito.anyObject());
 
+    File tmpDir = Files.createTempDir();
+    tmpDir.deleteOnExit();
+    this.jobConfDir = tmpDir.getAbsolutePath() + "/" + this.getClass().getSimpleName() + "/jobCatalog";
```

Comment:
Is it important to have a class name in this path? AFAIK, createTempDir will return a unique path on each call. (same in other test classes below)